control-plane/stor-port/src/types/v0/transport/replica.rs

```diff
@@ -556,6 +558,40 @@ impl From<ReplicaOwners> for models::ReplicaSpecOwners {
     }
 }
 
+/// Request struct to set replica entity id.
+#[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq)]
+pub struct SetReplicaEntityId {
+    pub uuid: ReplicaId,
+    pub entity_id: VolumeId,
+    pub node_id: NodeId,
```

Comment:
We have getters, so no need to make these pub

```suggestion
    uuid: ReplicaId,
    entity_id: VolumeId,
    node_id: NodeId,
```